index.html

```diff
@@ -67,6 +67,23 @@
 
         ga('create', gaID, 'auto');
         ga('send', 'pageview');
+
+        //Scroll to the top of the page.
+        window.onscroll = function() {scrollFunction()};
+
+        function scrollFunction() {
+        if (document.body.scrollTop > 20 || document.documentElement.scrollTop > 20) {
+        document.getElementById("myBtn").style.display = "block";
+         } else {
+        document.getElementById("myBtn").style.display = "none";
+         }
+         }
+
+// When the user clicks on the button, scroll to the top of the document
+        function topFunction() {
+        document.body.scrollTop = 0; // For Safari
+        document.documentElement.scrollTop = 0; // For Chrome, Firefox, IE and Opera
```

Comment:
The code in this file isn't deployed. This is only used during development time. This code should be placed in the `SearchGrid.vue` file, with the styles